distributed/worker.py

```diff
@@ -2658,7 +2675,7 @@ def _transition(
             return {}, []
 
         start = ts.state
-        func = self._transitions_table.get((start, cast(str, finish)))
+        func = self.TRANSITIONS_TABLE.get((start, cast(TaskStateState, finish)))
```

Comment:
nit: Should we generally keep the `_` around to mark the transition tables as internal use only, i.e. use `_TRANSITIONS_TABLE` in both `Scheduler` and `Worker`?

Reply:
sure